src/hooks/useInitializePage.ts

```diff
@@ -43,6 +45,7 @@ export const useInitializePage = () => {
           if (hiddenDuration >= RECONNECT_AFTER_HIDDEN_THRESHOLD) {
             // reconnect (reestablish connections to indexer, validator etc.) if app was hidden for more than 10 seconds
             IndexerWebsocketManager.getActiveResources().forEach((r) => r.restart());
+            CompositeClientManager.getActiveResources().forEach((r) => r.refreshConnections());
```

Comment:
so if i'm following correctly, we only want to reset the composite client due to the possibility a different validator will be better suited when returning to the tab?